roofit/hs3/src/JSONFactories_RooFitCore.cxx

```diff
@@ -309,6 +341,35 @@ class RooRealSumPdfStreamer : public RooJSONFactoryWSTool::Exporter {
 };
 } // namespace
 
+#include <RooRealSumFunc.h>
+
+namespace {
+class RooRealSumFuncStreamer : public RooJSONFactoryWSTool::Exporter {
+public:
+   std::string const &key() const override
+   {
+      const static std::string keystring = "sumfunc";
+      return keystring;
+   }
+   bool exportObject(RooJSONFactoryWSTool *, const RooAbsArg *func, JSONNode &elem) const override
+   {
+      const RooRealSumFunc *pdf = static_cast<const RooRealSumFunc *>(func);
+      elem["type"] << key();
+      auto &samples = elem["samples"];
```

Comment:
Why is this called `samples`? This is specific to HistFactory, but in general the functions can also represent things that are not "samples". Can it just be called "functions" without any compatibility issues?

Reply:
I opted for "samples" rather than "functions" because the structure of the sumpdf with "functions" and "coefficients" is clearly intended to work that way - if you're just adding random stuff, you could also use RooAddition or RooAddPdf